src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -525,7 +525,7 @@ private static ImmutableArray<RefKind> GetRefKindsOrNull(ArrayBuilder<RefKind> r
             for (int a = 0; a < rewrittenArguments.Length; ++a)
             {
                 BoundExpression argument = rewrittenArguments[a];
-                int p = (!argsToParamsOpt.IsDefault) ? argsToParamsOpt[a] : a;
+                int p = (!argsToParamsOpt.IsDefaultOrEmpty) ? argsToParamsOpt[a] : a;
```

Comment:
Why did this change? Default mapping was used to indicate that we do not have any remapping (common case). Otherwise remapping must match the length of parameters and it gets more complicated.